lib/evmone/instructions.hpp

```diff
@@ -938,6 +938,12 @@ inline constexpr auto callcode = call_impl<OP_CALLCODE>;
 inline constexpr auto delegatecall = call_impl<OP_DELEGATECALL>;
 inline constexpr auto staticcall = call_impl<OP_STATICCALL>;
 
+template <Opcode Op>
+Result newcall_impl(StackTop stack, int64_t gas_left, ExecutionState& state) noexcept;
```

Comment:
```suggestion
Result call2_impl(StackTop stack, int64_t gas_left, ExecutionState& state) noexcept;
```

Reply:
I was thinking about that too, not sure which is better, especially since we have `OP_CALL2`.